### ISSUE TITLE
Bug: Press FAB to add new task, the previously opened task is opened rather than the new task

### ISSUE THREAD

**Issue by XYIheng** [mpcjanssen/simpletask-android#1230]
### Description
When I want to add a new task, the previously opened task is opened, rather than the new task fragment.

### Step to reproduce
1. Open an existing task, e.g., 'A'
2. Click the options in the toolbar
3. Click 'Help'
4. Click the back button on the left side of the 'Help'. Then app returns to the task list rather than task A
6. Click back
7. Click the FAB to add a new task
8. See the error: The task 'A' is opened rather than a new task.

### Bug video

https://github.com/mpcjanssen/simpletask-android/assets/31406320/081c800e-5144-4614-86c8-9b83b6547e13

### Environment:
Simpletask: 11.0.1 (Cloudless Version from F-Droid)
Android8.0

**Comment by aubreyz** [comment]
I haven't tried to reproduce, and not sure why you are pressing help in the middle of stuff. However I don't think it is opening the existing task at all (your task has no text at all). I think it is using settings prepopulated with an existing filter. Try to do it with an actual task with some content, and slow the video down so we can see what is going on...

**Comment by XYIheng** [comment]
@aubreyz The task has the text 'A'. The must be something wrong when backing from the help page, as it affects the functionality of FAB

**Comment by XYIheng** [comment]
@aubreyz 
https://github.com/mpcjanssen/simpletask-android/assets/31406320/7ba375b3-76f7-4e63-8d65-2cbb10e9a623

I update a new video to explain this bug.

There are two tasks.
If I click the first task and then click the help button. Then back to the task list. If I click the FAB, it would redirect to the first task instead of creating a new task.

"not sure why you are pressing help in the middle of stuff "
Because this step is required to trigger this bug

"I think it is using settings prepopulated with an existing filter. "  
No, I didn't use any filter here

**Comment by aubreyz** [comment]
I agree that looks slightly odd - after a hell of a convoluted set of steps. 
Unfortunately given the behaviour of Google it seems that these sorts of things are never going to be fixed or explored unless someone takes over this project (which is the best todo.txt app there is on any platform .. so it is a real pity).
